redis/tests/test_cluster_async.rs

```diff
@@ -1758,7 +1759,7 @@ fn test_async_cluster_periodic_checks_update_topology_after_failover() {
     block_on_all(async move {
         let mut connection = cluster.async_connection().await;
         let mut prev_master_id = "".to_string();
-        let max_requests = 10000;
+        let max_requests = 5000;
```

Comment:
minor: why?

Reply:
We don't need to have so many tries when we have sleep waits in between calls